src/control/cmd/dmg/pool.go

```diff
@@ -160,6 +161,31 @@ func (d *PoolDestroyCmd) Execute(args []string) error {
 	return err
 }
 
+// PoolEvictCmd is the struct representing the command to evict a DAOS pool.
+type PoolEvictCmd struct {
+	logCmd
+	ctlInvokerCmd
+	// TODO: implement --sys & --svc options (currently unsupported server side)
```

Comment:
Seems like it would be better to just add them now so that the tool/request definitions don't need to be updated later.

Reply:
Am I correct in thinking that svc is referring to the pool replica service, and not some other service?
If so then should the interface for this just be the service rank?

Reply:
In the daos tool --svc was always the pool replica service ranks, so I assume that still follows here.

Reply:
I think your correct that we don't need it anymore. It looks like it might have just been necessary /relevant to the old DMG tool.